test/unit/dashboard/run-events.test.js

```diff
@@ -0,0 +1,131 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+// unit to test
+const handlers = require('../../../dashboard/server/api/runApi');
```

Comment:
This file should be `test/unit/dashboard/server/api/runApi.test.js`.

Related issue: we should follow a single convention for file names. either `runApi.js` (camelcase) or `frontent-events.js` (kebab-case).

Reply:
CamelCase is the way to go, fixed the changes!

Reply:
Let's create an issue to change all files to camelCase in that case (pun intended).

Reply:
https://github.com/elit-altum/newman-dashboard/issues/23